iceoryx_utils/test/moduletests/test_cxx_vector.cpp

```diff
@@ -899,3 +899,138 @@ TEST_F(vector_test, ConstructorWithSizeParameterGreaterThanCapacity)
     EXPECT_THAT(vector_test::copyCTor, Eq(5));
     ASSERT_THAT(sut.size(), Eq(5));
 }
+
+TEST_F(vector_test, TwoEmptyVectorOfSameCapacityAreEqual)
+{
+    vector<int, 10> a, b;
+
+    EXPECT_TRUE(a == b);
+    EXPECT_FALSE(a != b);
+}
+
+TEST_F(vector_test, TwoEmptyVectorOfDifferentCapacityAreEqual)
+{
+    vector<int, 10> a;
+    vector<int, 20> b;
+
+    EXPECT_TRUE(a == b);
+    EXPECT_FALSE(a != b);
+}
+
+TEST_F(vector_test, TwoEqualVectorsWithSameCapacityAreEqual)
+{
+    vector<int, 10> a, b;
+    a.emplace_back(1);
+    a.emplace_back(2);
+    a.emplace_back(3);
+
+    b.emplace_back(1);
+    b.emplace_back(2);
+    b.emplace_back(3);
+
+    EXPECT_TRUE(a == b);
+    EXPECT_FALSE(a != b);
+}
+
+TEST_F(vector_test, TwoEqualVectorsWithDifferentCapacityAreEqual)
+{
+    vector<int, 10> a;
+    a.emplace_back(4);
+    a.emplace_back(5);
+    a.emplace_back(6);
+
+    vector<int, 20> b;
+    b.emplace_back(4);
+    b.emplace_back(5);
+    b.emplace_back(6);
+
+    EXPECT_TRUE(a == b);
+    EXPECT_FALSE(a != b);
+}
+
+TEST_F(vector_test, TwoNonEqualVectorsWithSameCapacityAreNotEqual)
```

Comment:
shouldn't this be TwoVectorsWithDifferentSizeAreNotEqual?

Reply:
done